pyzabbix/api.py

```diff
@@ -261,6 +261,7 @@ def do_request(self, method, params=None):
         req = urllib2.Request(self.url, data)
         req.get_method = lambda: 'POST'
         req.add_header('Content-Type', 'application/json-rpc')
+        req.add_header('User-Agent', ' Mozilla/5.0 (Windows NT 6.1; WOW64; rv:12.0) Gecko/20100101 Firefox/12.0')
```

Comment:
Thank you for your contribution.

I think we should change user agent to `py-zabbix/<< __version__>>` and rise this pr to release branch instead.